pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -277,16 +281,15 @@ let
     bazelTarget = "//tensorflow/tools/pip_package:build_pip_package //tensorflow/tools/lib_package:libtensorflow";
 
     removeRulesCC = false;
+    # Without this Bazel complaints about sandbox violations.
+    dontAddBazelOpts = true;
 
     fetchAttrs = {
-      # So that checksums don't depend on these.
-      TF_SYSTEM_LIBS = null;
```

Comment:
Why was this removed? I recall having issues trying to remove this in the past working on the ROCm TensorFlow derivation, but don't remember what those issues were. It might just be here to prevent changes to inputs into this derivation from changing these checksums.

Reply:
> Why was this removed?

The way TensorFlow switches between the system and bundled dependencies is that at fetch time it either downloads the dependency and creates a repository out of it, or (if the dependency is listed in `TF_SYSTEM_LIBS`) creates the "proxy" repository that points to the system library using the files bundled with the "main" TensorFlow source code, see https://github.com/tensorflow/tensorflow/tree/master/third_party/systemlibs.

This means that if `TF_SYSTEM_LIBS` is not set at fetch time, the build won't use the system libraries, so having it set in any section outside of "fetch" is basically a no-op.

> I recall having issues trying to remove this in the past working on the ROCm TensorFlow derivation

Given the above, in the derivation currently present in Nixpkgs, the build doesn't actually use any of the system libraries, so the build that uses system libraries was likely never tested before and enabling them uncovered several issues - like incorrect dependencies or incompatibilities between the libraries bundled with TF and provided in Nixpkgs (see the comments for "protobuf" and "re2" in the changelist). It also could very well be that there are ROCm-specific issues on top of this, though.

> It might just be here to prevent changes to inputs into this derivation from changing these checksums.

Given that these "proxy" repositories that point to system libraries shouldn't encode any version-specific information (normally they just encode the library name to link to, e.g. see https://github.com/tensorflow/tensorflow/blob/master/third_party/systemlibs/jsoncpp.BUILD) - I don't expect that changing the inputs versions will change the checksum of the dependencies?